cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -97,9 +98,10 @@ def test_provision_from_template(provider, provisioned_instance):
 @pytest.mark.provider([GCEProvider], required_fields=[['provisioning', 'image']],
                       override=True)
 @pytest.mark.usefixtures('setup_provider')
+@pytest.mark.meta(blockers=[BZ(1619298, forced_streams=['5.9', '5.10'])])
```

Comment:
This isn't blocking on 59 in PRT, and the test fails with `GCE Instance isn't Preemptible (cfme/tests/cloud_infra_common/test_provisioning.py:110)`

Reply:
@mshriver could you please fix the #7661 to unblock this PR? Thanks.